internal/awsiotprotocol/mqtts_test.go

```diff
@@ -30,9 +30,9 @@ func TestMqttsNewClientOptions(t *testing.T) {
 	opt := &Config{
 		URL:      "mqtts://example.com:8882",
 		ClientID: "mqttsclientid",
-		CaPath:   "../../sample/samplecerts/cafile.pem",
-		CertPath: "../../sample/samplecerts/client-crt.pem",
-		KeyPath:  "../../sample/samplecerts/client-key.pem",
+		CaPath:   "../../examples/mqtt/samplecerts/cafile.pem",
+		CertPath: "../../examples/mqtt/samplecerts/client-crt.pem",
+		KeyPath:  "../../examples/mqtt/samplecerts/client-key.pem",
```

Comment:
Sorry, these files are used only by this test. `samplecerts` seems better to be moved to this directory.

Reply:
OK. Makes sense.